### DISCUSSION TITLE
Colorscheme

### DISCUSSION THREAD

**Discussion by oryx-exe** [LazyVim/LazyVim#4598] [Q&A] [answered]
I am struggling to install the cyberdream colorscheme and need some help any pointers
![image](https://github.com/user-attachments/assets/292d40ea-7b81-465a-bcc0-cc58ab910893)
This is what ive got so far

**Comment by dpetka2001** [comment, marked as answer]
```lua
return {
  {
    "scottmckendry/cyberdream.nvim",
    opts = {},
  },
  {
    "LazyVim/LazyVim",
    opts = {
      colorscheme = "cyberdream",
    },
  },
}
```
should do it I believe. It's also mentioned in the [docs](https://www.lazyvim.org/configuration#icons--colorscheme) how to change colorscheme. All the configuration options of cyberdream should go inside the `opts` in the cyberdream plugin spec.

**Reply by oryx-exe** [reply]
Thank you so much, you are goated, I think it worked, do you know anything about icons, mine look a little funny

**Reply by oryx-exe** [reply]
![Screenshot from 2024-10-16 18-46-13](https://github.com/user-attachments/assets/2fed17ce-a48e-4f8e-b683-d6e9ec2a24a5)

**Reply by dpetka2001** [reply]
You have to install [nerdfonts](https://www.nerdfonts.com/) for icons and also configure your terminal to use them.
